nrich-form-configuration-spring-boot-starter/README.md

```diff
@@ -18,62 +21,78 @@ The artifact is published on [Maven Central Repository](https://search.maven.org
+readability):
 
-| property                   | description                                                                                                                                                                                            | default value |
-|----------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|
-| default-converter-enabled  | Whether default converter service `DefaultConstrainedPropertyValidatorConverterService` for converting `ConstrainedProperty` instances to `ConstrainedPropertyClientValidatorConfiguration` is enabled | true          |
-| form-configuration-mapping | Mapping between a client side form identifier and class holding the constraints for the form (usually the class accepted as input on the server side)                                                  |               |
+| property                   | description                                                                                                                                                                                                                                                                                                                                                   | default value |
+|----------------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|
+| default-converter-enabled  | whether default converter service [`DefaultConstrainedPropertyValidatorConverterService`][default-constrained-property-validator-converter-service-url] for converting [`ConstrainedProperty`][constrained-property-url] instances to [`ConstrainedPropertyClientValidatorConfiguration`][constrained-property-client-validator-configuration-url] is enabled | true          |
+| form-configuration-mapping | mapping between a client side form identifier and class holding the constraints for the form (usually the class accepted as input on the server side)                                                                                                                                                                                                         |               |
 
-The default configuration values in yaml format for easier modification are given bellow:
+The default configuration values are given bellow in a yaml format for easier modification:
 
 ```yaml
-
 nrich.form-configuration:
```

Comment:
I think yaml property indentation is 2 spaces in our code style. Did you reformat it or are we missing some editorconfig? (Check also other PR's)

Reply:
I reformated it with editorconfig and this is how it turned out.

Reply:
Strange, we have the configuration for yaml formatting to 2 spaces and even if I try to add it to MD to have two spaces it just reformats with 4. Maybe it's a bug in intellij formatter. Not sure what to do here, since if you change only a part of files it won't be consistent with the rest. If you will reformat this then we need to reformat all the MD files that have yaml, I am fine either way.

Reply:
Yes, I am also not sure what to do about it. Maybe your solution is the best option.